client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```
@@ -123,16 +123,22 @@ $deisgn-selector-selection-space: 175px;
 .page-layout-selector__grid {
 	display: grid;
 	grid-gap: 1.25em;
-	@include breakpoint( '>660px' ) {
-		grid-template-columns: 1fr 1fr 1fr;
-	}
+	grid-template-columns: repeat( auto-fill, minmax( 280px, 1fr ) );
 }
 
 .page-layout-selector__item {
 	position: relative;
 	overflow: hidden;
 	padding: 2px;
 
+	&:before {
```

Let's use `::` pseudo-element selector:

```suggestion
	&::before {
```